### ISSUE TITLE
debuffs: duration calculated wrong when same debuff is applied to different targets of the same name

### ISSUE THREAD

**Issue by Grookshank** [shagu/pfUI#627]
If you apply the same debuff (e.g. Curse of the Elemets) to a target (e.g. Scarshield Sentry), the target dies one minute later and then the same debuff is applied to another Scarshield Sentry, then the duration starts at the duration left on the first target when it died.

**Comment by shagu** [comment]
This is related on how a debuff timers is working for Vanilla. Let me try to explain this:
The Vanilla Wow API doesn't provide any function that enables me to read any remaining debuff times for enemies. So the addon has to scan the combat log for messages that indicate whenever a debuff has been applied to an enemy and then query its internal spell database (https://github.com/shagu/pfUI/blob/master/env/locales_enUS.lua#L1445) to retrieve the default time that debuff usually has. There aren't any UUIDs and the combat log only contains the `name` and the `spell`.
tl;dr: It's impossible for me to distinguish between two mobs if they share the same name.
